components/table/test/table.test.js

```diff
@@ -29,3 +30,17 @@ describe('d2l-table-wrapper', () => {
 	});
 
 });
+
+describe('d2l-table-header', () => {
+
+	it('should construct', () => {
+		runConstructor('d2l-table-header');
+	});
+
+	// it('should override default SelectionHeader label', async() => {
+	// 	const el = await fixture(html`<d2l-table-header></d2l-table-header>`);
+	// 	const section = el.shadowRoot.querySelector('section');
+	// 	expect(section.getAttribute('aria-label')).to.equal('Table controls');
+	// });
```

Comment:
@svanherk I've added this in to match the testing in your PR.

My plan is to actually leave this as commented for this PR, and open another PR afterward to uncomment this test and add langterms etc. That way this PR won't get even bigger 😄.